src/Microsoft.Health.Fhir.Shared.Api/Controllers/ConvertDataController.cs

```diff
@@ -180,6 +189,7 @@ private static HashSet<string> GetSupportedParams()
                 ConvertDataProperties.InputDataType,
                 ConvertDataProperties.TemplateCollectionReference,
                 ConvertDataProperties.RootTemplate,
+                ConvertDataProperties.JsonDeserializationTreatDatesAsStrings,
```

Comment:
By adding it in the supported param list, upon validation of params we are enforcing this parameter is provided in the request, right? Is that intended or can this simply not be provided in the request and default if so. If it is the former, I suppose we will need an api version update.

Reply:
The supported param list contains all of the params that this controller works with. So if a parameter provided by the end user doesn't exist within this list then the user will get an error. It doesn't enforce that these params need to _exist_, though. The various `ReadXXXParameter` method seem to do that.  

As you've stated, I've opted to allow a default value to be used if one isn't provided. I added a new `ReadBoolParameter` which will return `false` if the parameter is not present.